sw/device/lib/testing/test_coverage.c

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/uart.h"
+#include "sw/vendor/llvm_clang_rt_profile/compiler-rt/lib/profile/InstrProfiling.h"
+#include <stdint.h>
+
+/**
+ * When the linker finds a definition of this symbol, it knows to skip loading
+ * the object which contains the profiling runtime’s static initializer.
+ */
+int __llvm_profile_runtime;
```

Comment:
Is this symbol LLVM magic? Can you describe who is actually checking for this symbol?

Reply:
I thought the comment was descriptive enough. Please also see [this section](https://clang.llvm.org/docs/SourceBasedCodeCoverage.html#using-the-profiling-runtime-without-static-initializers) in the first reference.

Reply:
Can you just include a link to that in this file? "The linker" is really vague in a "is GNU ld somehow figuring this out?" way.